websites/I/Incels.is/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "351653707496292353",
+		"name": "ianhell666"
+	},
+	"service": "Incels.is",
+	"description": {
+		"en": "Incels.is is a community for men that struggle with or are unable to get into romantic relationships with women despite trying. We welcome men from all walks of life, and from all cultural and racial backgrounds, as long as you are an incel."
+	},
+	"url": [
+		"incels.is",
+		"www.incels.is"
+	],
+	"version": "1.0.0",
+	"logo": "https://incels.is/logo/512x512.png",
+	"thumbnail": "https://incels.is/logo/512x512.png",
```

Comment:
Thumbnail should be a banner image or a screenshot of the website homepage.

Reply:
Added a 16:9 FHD banner image.